packages/xrpc-server/src/server.ts

```diff
@@ -47,6 +67,18 @@ export class Server {
     }
   }
 
+  // @TODO redesign this api, streaming should not have to be enabled
+  enableStreaming(server: http.Server) {
+    server.on('upgrade', async (req, socket, head) => {
+      const url = new URL(req.url || '', 'http://x')
+      const sub = this.subscriptions.get(url.pathname.replace('/xrpc/', ''))
```

Comment:
This isn't exactly a bug, but it's also not working for the right reason.  We want the URL to look like `/xrpc/{nsid}`, but there's nothing here really checking that the pathname starts with `/xrpc`.  It happens to work because `{nsid}` on its own isn't a valid pathname since it doesn't start with a `/` 🙃